src/interactive/Dropdown.jsx

```diff
@@ -0,0 +1,114 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import cx from 'classnames';
+import bindAll from '../utils/bindAll';
+
+/**
+ * @module Dropdown
+ */
+class Dropdown extends React.Component {
```

Comment:
We'll need the `componentDidMount` and `componentWillUnmount` lifecycle methods here to handle dropdown dismissal when the user clicks outside the dropdown content. I'll mark this with `Prelim` until that change is made.